crates/floresta-chain/src/pruned_utreexo/chain_state.rs

```diff
@@ -715,7 +715,8 @@ impl<PersistedState: ChainStore> ChainState<PersistedState> {
         inner.best_block.best_block = best_block;
         inner.best_block.depth = height;
     }
-    fn verify_script(&self, height: u32) -> bool {
+    /// Returns if assume_valid is true.
+    fn validate_signatures(&self, height: u32) -> bool {
```

Comment:
I see no reason to verify a script without it being for signature validation... thats why the rename.